sass-graph/sass-graph.d.ts

```diff
@@ -0,0 +1,81 @@
+// Type definitions for sass-graph
```

Comment:
Oh sorry, I should've seen that earlier. The version is now added. Thanks for the quick review!